tasks/src/test/java/webdriver/google_cloud/browser/Browser.java

```diff
@@ -0,0 +1,26 @@
+package webdriver.google_cloud.browser;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class Browser {
+  private final static int TIME_OUT_IN_SECONDS = 100;
+
+  public static WebElement waitForVisibility(WebDriver driver, WebElement element) {
```

Comment:
I'd get rid of a driver passing to the method and just call DriverSingleton.getDriver() inside the method